### ISSUE TITLE
Unable to achieve 4K output

### ISSUE THREAD

**Issue by seancafferky** [mayankk2308/purge-wrangler#124]
### System Configuration
Describe your system configuration here. State any anomalies (USB drives, etc.) involved as well.
- **Mac**: MacBook Pro 15" Retina Mid-2015
- **macOS Version**: 11.0.1
- **System GPUs**: Intel Iris & Radeon RX 580
- **eGPU**: Razor Core X
- **Script Version**: The version available on Nov 29, 2020 ...which seems to be version 6.3.3, apparently.
- **Script Logs**: Attach logs generated by the script.

You can generate logs from the script by choosing **More Options** > **System Diagnosis & Logging**.

### Issue Description
When connecting to a 4K TV (e.g., TCL 55S517), the MacOS System Information profiler reports the resolution as only being 1920 x 1080 (1080p FHD - Full High Definition)... whereas I'm expecting 4K. 

That was the only point of even connecting an eGPU and setting up purge-wrangler in the first place: to get 4K output. Otherwise, without an eGPU, the crappy onboard Intel Iris Pro can already output 1080p to the same display without a need for any of this extra effort.

### Expected Behavior
I want to achieve 4K output.

### Screenshots
[pwlog-2020-11-29-18-50-41.zip](https://github.com/mayankk2308/purge-wrangler/files/5613547/pwlog-2020-11-29-18-50-41.zip)
![Screen Shot 2020-11-29 at 6 59 34 PM](https://user-images.githubusercontent.com/3267299/100564177-146a3800-3275-11eb-9840-8e0947fc7e61.jpg)
![Screen Shot 2020-11-29 at 7 24 58 PM](https://user-images.githubusercontent.com/3267299/100565493-9e67d000-3278-11eb-8c3c-c88a706748b1.jpg)

**Comment by mayankk2308** [comment]
This has nothing to do with the script. Problem is either with TV settings or macOS compatibility with TV. I have successfully run 2 4K monitors at 60Hz on a 2014 15" MBP GT750M without an eGPU (as reference) in the past.

**Comment by seancafferky** [comment]
Thank you for a challenging response. What you had written made no sense to me, in that the TV works just fine in 4K and there's nothing wrong with macOS either, so I pressed on despite the flippant dismissal. After further investigations, I deduced that one HDMI cable in the longer daisy chain was not 4K compatible and replaced it with successful results. I thought I would share this outcome in hopes that it may help someone else in the future who is trying to pinpoint areas to troubleshoot.
